### ISSUE TITLE
HTML does not validate

### ISSUE THREAD

**Issue by juleskuehn** [aaact-aatia/a11y-req#19]
There are numerous issues with aria roles, nesting of elements etc. which would fail WCAG. In all there are 25 issues,

**Comment by juleskuehn** [comment]
Note that this issue cannot be resolved without upstream changes to WET.
e.g. "ARIA role tabpanel must be removed when the element is made visible, as it is not allowed for the element"
This code is added by [WET Tabbed Interface](https://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html):
`<details id="details-step1" open="open" class="wb-auto-2-grp fade out noheight" role="tabpanel" aria-hidden="true" aria-expanded="false" aria-labelledby="details-step1-lnk">`
